test/timer.cpp

```diff
@@ -76,7 +76,11 @@ BOOST_AUTO_TEST_CASE(timer_start_stop)
 
     BOOST_REQUIRE_EQUAL(tick_count, required_tick_count);
 
+#ifdef WIN32
+    BOOST_REQUIRE_LE(abs(test_duration - required_duration), milliseconds(16*required_tick_count));
+#else
     BOOST_REQUIRE_LE(abs(test_duration - required_duration), milliseconds(10));
+#endif
```

Comment:
I think I've seen this to be a problem on other platforms as well, or when I was using valgrind.